app/controllers/application_controller.rb

```diff
@@ -180,7 +181,8 @@ def set_raven_context
   # event api.
   def user_is_authenticated?
     controllers = %w[passwords sessions]
-    paths = [new_user_registration_path, hardware_items_path(:json), events_path(:json)]
+    #paths = [new_user_registration_path, hardware_items_path(:json), events_path(:json)]
```

Comment:
I mentioned this in the issue. I'm not sure how to avoid this - since the token auth is in a different namespace, can I point it away from this controller/this functionality?